api/http/api_deployments.go

```diff
@@ -1066,6 +1066,39 @@ func (d *DeploymentsApiHandlers) GetDeploymentStats(w rest.ResponseWriter, r *re
 	d.view.RenderSuccessGet(w, stats)
 }
 
+func (d *DeploymentsApiHandlers) GetMultipleDeploymentStats(w rest.ResponseWriter, r *rest.Request) {
```

Comment:
Nitpicking: I don't like "GetMultipleDeploymentStats", I believe "GetDeploymentsStats" is less wordy.